### PR TITLE
boyscout & consolidate menu items

### PR DESCRIPTION
**Pull request by fabiangotzen**
Most of it is just fixing TSLint errors, I've also tried to simplify the menu: 

image

So you directly navigate to the list component. From there you have the posibility to add the flight/glider/place: 
image
image
image

### CODE REVIEW

File: src/app/account/register/register.page.ts [hunk 1]
```diff
@@ -47,7 +48,7 @@ export class RegisterPage implements OnInit, OnDestroy {
       },
         (async (error: any) => {
           await loading.dismiss();
-          if (error.status === 409) {
+          if (error.status === 4) {
```
Comment (laggery) [inline src/app/account/register/register.page.ts:51]: What is the error code 4?

File: src/app/glider/glider-list/glider-list.page.ts [hunk 2]
```diff
@@ -35,25 +43,27 @@ export class GliderListPage implements OnInit, OnDestroy, AfterViewInit {
...
-    this.gliderService.getGliders({ limit: this.gliderService.defaultLimit, clearStore: true }).pipe(takeUntil(this.unsubscribe$)).subscribe(async (res: Glider[]) => {
-      // @hack for hide export item
-      setTimeout(async () => {
-        this.content.scrollToPoint(0, 48);
+    this.gliderService.getGliders({ limit: this.gliderService.defaultLimit, clearStore: true })
+      .pipe(takeUntil(this.unsubscribe$))
+      .subscribe(async (res: Glider[]) => {
+        // @hack for hide export item
+        setTimeout(async () => {
+          this.content.scrollToPoint(0, 48);
```
Comment (laggery) [inline src/app/glider/glider-list/glider-list.page.ts:61]: Maybe also add await like in flight-list?

File: src/app/place/place-list/place-list.page.ts [hunk 3]
```diff
@@ -36,11 +36,13 @@ export class PlaceListPage implements OnInit, OnDestroy, AfterViewInit {
...
       message: this.translate.instant('loading.loading')
     });
     await loading.present();
-    this.placeService.getPlaces({ limit: this.limit, clearStore: true }).pipe(takeUntil(this.unsubscribe$)).subscribe(async (res: Place[]) => {
+    this.placeService.getPlaces({ limit: this.limit, clearStore: true })
+      .pipe(takeUntil(this.unsubscribe$))
+      .subscribe(async (res: Place[]) => {
       // @hack for hide export item
       setTimeout(async () => {
         this.content.scrollToPoint(0, 48);
```
Comment (laggery) [inline src/app/place/place-list/place-list.page.ts:48]: Maybe also add await like in flight-list?

**Review by laggery — COMMENTED** [review]
Design bug:
image

The "add" button is at the very end of the list. To do this I first have to load 1000 flights
Same for place list and glider list

File: src/app/account/register/register.page.ts (hunk 1 above)
Comment (fabiangotzen) [reply src/app/account/register/register.page.ts]: typo, fixed:  HttpStatusCode.CONFLICT

File: src/app/glider/glider-list/glider-list.page.ts (hunk 2 above)
Comment (fabiangotzen) [reply src/app/glider/glider-list/glider-list.page.ts]: done

File: src/app/place/place-list/place-list.page.ts (hunk 3 above)
Comment (fabiangotzen) [reply src/app/place/place-list/place-list.page.ts]: done

**Comment by fabiangotzen** [comment]
> Design bug:
> image
> 
> The "add" button is at the very end of the list. To do this I first have to load 1000 flights
> Same for place list and glider list

I took some inspiration from the android add contacts list: (moved filter there as well)

image

let me know what you think.

**Comment by fabiangotzen** [comment]
Also, a small redesign of the statistics page: 
image
(increased font size + put them into ion-cards)

**Comment by laggery** [comment]
> Also, a small redesign of the statistics page:
> image
> (increased font size + put them into ion-cards)

Design:
- The idea to have bigger font is good but when you have a lot of flights it cause some design problem
- For me, the filter is not on the right place because it also filter the graphics
- Card on statistics make the graphics lot smaller on mobile and I find we lose to much places
image

About "add" and "Filter" button.
Filter button -> in my opinion not clear enough that it is the filter button. It is also not anytime accessible. For example when i scroll down in my flights and then I want to change the filter I need first to scroll up
The add button is small but takes a lot of place so. I don't have a better solution at the moment.

**Comment by fabiangotzen** [comment]
> Design:
> 
> * The idea to have bigger font is good but when you have a lot of flights it cause some design problem
> * For me, the filter is not on the right place because it also filter the graphics
> * Card on statistics make the graphics lot smaller on mobile and I find we lose to much places
>   image
> 
> About "add" and "Filter" button.
> Filter button -> in my opinion not clear enough that it is the filter button. It is also not anytime accessible. For example when i scroll down in my flights and then I want to change the filter I need first to scroll up
> The add button is small but takes a lot of place so. I don't have a better solution at the moment.

* Adjusted the statistics page: 
image
*reverted changes to filter and added the "add" buttons to the footer which is always there.
image

**Comment by fabiangotzen** [comment]
image
moved it to the top

**Pull request merged**